src/lib/pay-request/services/admin_users/client.ts

```diff
@@ -132,6 +134,21 @@ export default class AdminUsers extends Client {
         .then(response => client._unpackResponseData<User>(response));
     },
 
+    updateFeatures(
+        id: string,
+        operation: FeatureOperation,
+        value: string
+    ): Promise<User | undefined> {
+      const payload = {
+        path: 'features',
+        op: operation,
+        value
+      }
+      return client._axios
+          .patch(`/v1/api/users/${id}`, payload)
+          .then(response => client._unpackResponseData<User>(response));
```

Comment:
should this not also use the `User` constructor?

Reply:
not that it matters much since it seems the return value isn't used anywhere

Reply:
Since I've changed the User from an interface to a class with a constructor, I think everywhere that's doing `.then(response => client._unpackResponseData<User>(response));` should be superseded by `.then(response => new User(response.data));` now?

Reply:
Yeah I think so, it should be straightforward to do at least